prisma/seed.ts

```diff
@@ -428,3 +433,29 @@ function getNetworks() {
       }
   ]
 }
+
+function getBadges() {
```

Comment:
For preprod and prod. Real data will be needed to be loaded in the database automatically.

Reply:
what is the procedure to manage like for networks and projects ?

Reply:
The procedure is still to be defined by @cloudvenger and myself.